tests/e2e/playwright-quickstart.config.js

```diff
@@ -14,7 +14,8 @@ const config = {
         trace: 'on',
         baseURL: 'http://localhost:9000/#',
         ignoreHTTPSErrors: true,
-        myItemsFolderName: "My Items"
+        myItemsFolderName: "My Items",
+        failOnConsoleError: false
```

Comment:
Let's remove this so it's enabled by default and we can disable on a suite-by-suite basis